### DISCUSSION TITLE
How to run the windows binary?

### DISCUSSION THREAD

**Discussion by bvqbao** [ory/kratos#3918] [Q&A]
Hello,

I would like to run the windows binary. The documentation stops at downloading the binary and then the QuickStart section uses docker :').
From my understanding, you need a config file for Kratos but I can't find it. Basically, I would like to run Ory Kratos directly on WIndows without docker.

Is there any guide/tutorial somewhere?

Regards,
Bao Bui

**Comment by vinckr** [comment]
Hello @bvqbao 

There is no tutorial directly for Windows, but you can refer to this tutorial and adapt it to Windows, the approach should be the same: https://www.ory.sh/docs/kratos/guides/deploy-kratos-example
